lib/cog/commands/helpers.ex

```diff
@@ -133,6 +133,8 @@ defmodule Cog.Commands.Helpers do
     do: "Could not find '#{resource_type}' with the name '#{resource_name}'"
   def error(:invalid_handle),
     do: "Invalid chat handle"
+  def error({:trigger_invalid, %Ecto.Changeset{}=changeset}),
+    do: changeset_errors(changeset)
```

Comment:
My $0.02 would be to tackle any changing of return values or anything else later; breaking up the commands is a big enough change as it is. It'll also be easier to review if there's only one thing you're dealing with.